src/include/builddefs.in

```diff
@@ -175,7 +175,7 @@ ECHO	= @echo@
 LN_S	= @LN_S@
 GREP    = @grep@
 GIT	= @GIT@
-CLINT	= @CPPCHECK@ --std=posix --force --quiet --error-exitcode=1 -I$(TOPDIR)/src/include/pcp -I/usr/include/bits
+CLINT	= @CPPCHECK@ --library=posix --force --quiet --error-exitcode=1 -I$(TOPDIR)/src/include/pcp -I/usr/include/bits
```

Comment:
Have we checked how far back this change exists?  el6 era hosts? older debian distros? BSDs?
@kmcdonell any insight into if we'll need a check for this flag?

Reply:
Oops, I thought about it, but then forgot. What's a quick way to check this? Looking up the cppcheck source code, finding the version which introduced the `--library=posix` flag, and then check all 100 package managers and distro versions sounds a bit tedious :|

(The most obvious way would be to integrate all supported platforms into azure pipelines ;))